### ISSUE TITLE
Warning when add custom Mix task in project

### ISSUE THREAD

**Issue by 7sDream** [iampeterbanjo/vscode-elixir-linter#91]
1. create a project with mix
2. create folder `lib/mix/tasks`
3. add `start.ex` in `lib/mix/tasks` with content:

```elixir
defmodule Mix.Tasks.Start do
  use Mix.Task

  def run(_), do: "Hello World" |> IO.puts
end
```

Elixir Linter will show a warning on line 1: `[ElixirLS Dialyzer] Callback info about the 'Elixir.Mix.Task' behaviour is not available`

![warning](https://i.loli.net/2018/08/19/5b78459c4d882.png)

---

Append: After I follow [those steps on stackoverflow](https://stackoverflow.com/questions/51208388/how-to-fix-dialyzer-callback-info-about-the-behaviour-is-not-available), run `mix dialyzer` in shell don't show warnings anymore, but VS Code keep showing.

**Comment by 7sDream** [comment]
Sorry, the warning is issued by another extension.
